README.md

```diff
@@ -1,17 +1,53 @@
-# terraform-registry
-An implementation of the [Terraform Registry API][registry-api] using [Chalice][chalice] and [PynamoDB][pynamodb].
+# zeroae/terraform-registry
+Zero A.E.'s [12-Factor][12-factor] codebase of the [Terraform Registry API][registry-api] implemented using [Chalice][chalice] and [PynamoDB][pynamodb].
 
 ## Similar projects (and likely more complete), in alphabetical order
-  - [apparentlymart/terraform-simple-registry](https://github.com/apparentlymart/terraform-simple-registry)
   - [apparentlymart/terraform-aws-tf-registry](https://github.com/apparentlymart/terraform-aws-tf-registry)
+  - [apparentlymart/terraform-simple-registry](https://github.com/apparentlymart/terraform-simple-registry)
   - [dflook/terraform-registry](https://github.com/dflook/terraform-registry)
   - [outsideris/citizen](https://github.com/outsideris/citizen)
   - [rmb938/tf-registry](https://github.com/rmb938/tf-registry)
   
 ## Deployment on AWS
 
+## Development
+1. Requirements: 
+    - conda
+    - docker
```

Comment:
```suggestion
    - docker
    - docker-compose
```
It turns out that conda is not really a requirement for the instructions below.

Reply:
I'll keep it here anyways because it brings in terraform for running integration tests.